### DISCUSSION TITLE
Let user select "code-action" even if there's only one candidate.

### DISCUSSION THREAD

**Discussion by ianyepan** [emacs-lsp/lsp-mode#3428] [Q&A] [answered]
As titled, is there a variable (boolean t/nil) that controls this behavior? Currently, when there are more than one code action candidates at point, invoking lsp-execute-code-action will display the actions candidates in whichever completion framework the user uses (in my case, ivy/counsel). However, if there's only one candidate, lsp-mode will directly apply the code-action at point, without prompting the user to select.
I would like to select (and in the process, see the name of the code action), even if there's only one candidate. Any help is appreciated!

**Comment by yyoncho** [comment, marked as answer]
lsp-auto-execute-action

**Comment by ianyepan** [comment]
Thanks a ton, exactly what I was looking for.
